### ISSUE TITLE
Question about using model buddles. 

### ISSUE THREAD

**Issue by metaism-ding** [azavea/raster-vision#986]
Hi, 
thank you for sharing the trained buddles.
However, when I tried ISPRS Potsdam segementation buddle, I can not get the prediction.
The command is here and you can see the error message.
The buddle is downloaded form your example homepage. 
```
rastervision predict $buddle $image_uri $label_uri
```
ERROR:
```
FileNotFoundError: [Errno 2] No such file or directory: '/opt/data/tmp/tmp6ut8_2hy/tmpa7bi6qk2'
```
and 
```
rastervision.pipeline.file_system.file_system.NotReadableError: Could not read s3://raster-vision-raw-data/isprs-potsdam/4_Ortho_RGBIR/top_potsdam_2_10_RGBIR.tif
```
I just use local buddle and image. Can you tell me what is wrong with it?
The same command shows no problem when using other buddles.

Thanks.

**Comment by lewfish** [comment]
You're right -- this problem started occurring some time after the 0.12 release. I will try to find a fix. In the meantime, you should be able to use the 0.12 tag.

**Comment by lewfish** [comment]
Actually, 0.12 won't work because of a separate bug which was fixed after 0.12 in #980

**Comment by lewfish** [comment]
I fixed the issue in the linked PR.
